webapp/README.md

```
@@ -0,0 +1,54 @@
+
+Then open http://localhost:3000 and drop a PDF onto the page. The converted
+Markdown shows up below the upload box and you can copy it to the clipboard or
+download it as a `.md` file.
+
+Set `PORT=8080 npm start` to listen on a different port.
+
+## API
+
+The page uses a single endpoint:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)